tests/safeds/data/tabular/transformation/test_one_hot_encoder.py

```diff
@@ -210,6 +210,49 @@ def test_should_not_change_original_table(self) -> None:
 
         assert table == expected
 
+    def test_get_names_of_added_columns(self) -> None:
+        transformer = OneHotEncoder()
+        with pytest.raises(TransformerNotFittedError):
+            transformer.get_names_of_added_columns()
+
+        table = Table(
+            {
+                "a": ["b"],
+            },
+        )
+        transformer = transformer.fit(table, None)
+        assert transformer.get_names_of_added_columns() == ["a__b"]
```

Comment:
It would be usefull to add more values to the Table to test if the second for loop in line 265 (OneHotEncoder) works

Reply:
Added a more complicated testcase.